lineapy/api/api.py

```diff
@@ -437,13 +437,13 @@ def to_pipeline(
         artifacts=artifacts,
         name=pipeline_name,
         dependencies=dependencies,
-        input_parameters=input_parameters,
-        reuse_pre_computed_artifacts=reuse_pre_computed_artifacts,
     )
     pipeline.save()
     return pipeline.export(
         framework=framework,
         output_dir=output_dir,
+        input_parameters=input_parameters,
+        reuse_pre_computed_artifacts=reuse_pre_computed_artifacts,
         pipeline_dag_config=pipeline_dag_config,
     )
 
```

Comment:
There should be an equivalent get_pipeline that should need editing as well.

Reply:
Done and pushed!